.travis/run_ctest.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if ! ctest \
+    -output-on-failure \
```

Comment:
I think a dash is missing here:  `-output-on-failure` => `--output-on-failure`

Reply:
Resolved now in  c17e364